src/test/java/org/fundacionjala/coding/wilson/KataFizzBuzzTest.java

```diff
@@ -0,0 +1,75 @@
+package org.fundacionjala.coding.wilson;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class KataFizzBuzzTest {
+
+    final String fizzBuzz = "FizzBuzz";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)